### ISSUE TITLE
070 national agricultural library call number

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#89]
https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/0XX.csv

**Comment by cspayne** [comment]
@pennylenger @GordonDunsire 
The google sheet and document indicate different mappings for when 070 is a "serial-like item" versus not. How do I determine this?

**Comment by pennylenger** [comment]
I am not sure, but I looked at the example: 
070 | 0#$a281.9$bC81A [Number assigned from the USDA library scheme.]
070 | 0#$aQH301.A5 $b1981 [Number assigned from the LC classification scheme.]
Can we use a pattern like [A-Z][1-9] to determine the LC classification scheme?

**Comment by GordonDunsire** [comment]
@cspayne, @pennylenger: The transformation of 'serial-like' resources is out of scope for this phase. What determines 'serial-like' will be dependent on the values of other fields, especially 00x.

However, I agree with @pennylenger's suggestion. It avoids getting entangled in diachronic weeds, and it is a really neat way of distinguishing LC classification from other schemes.

**Comment by cspayne** [comment]
"Prefixes may occur in NAL call numbers. "A" specifies a USDA publication in the old USDA scheme; "a" specifies a USDA publication in the LC classification scheme; "R" specifies a rare book; "J" specifies a juvenile book; and "Fo" specifies a folio publication."

Do we know whether numbers assigned from the USDA library scheme are always only numbers or beginning with "A" and then only numbers? Or can we only determine whether it's from LC when it starts with "a"?

**Comment by GordonDunsire** [comment]
@cspayne: I don't know anything about the old USDA scheme, but in the example '281.9' cannot be based on any LC scheme because it is a decimal number without a preceding alphabetical class indicator. So presumably it is the same USDA classification as 'A 281.9'. This cannot be confused with LC numbers under class indicator 'A' because it is immediately sub-classed by a second upper-case alpha character, for example 'AC'.

So I think it's safe to remove 'A' if it is immediately followed by a numeric character, and treat it as being in the original USDA scheme.

It's also ok to remove the prefix 'a' and treat the value as being in the LC scheme.

I have no idea what to do with the other prefixes, or this example from the MARC 21 manual:

070 	0# $aTRANSL$b17828

**Comment by AdamSchiff** [comment]
Since we don't do anything with NAL numbers, I'm not familiar with them.

Adam

Adam L. Schiff
Principal Cataloger
University of Washington Libraries
(206) 543-8409
***@***.***
________________________________

**Comment by CECSpecialistI** [comment]
@CECSpecialistI will ask Amanda Xu

**Comment by CECSpecialistI** [comment]
Amanda's response:
 
"The classification of the Library of USD was made by W.I. Fletcher, the librarian of Amherst College.  It was expanded into 5th edition to cover the growth of the library collection.  Prior to 1965, NAL used a USDA library classification scheme. In 1965, NAL began using the Library of Congress Classification for all newly-cataloged materials. Serial-like items have retained the old call number whenever possible; thus, both schemes are still in use.  

For USDA publications, we are still using prefix “a” in front of LC Classification scheme. For juvenile book, we use prefix “J”.  For folio publication, we use “Fo”.  I did shelf listing and found prefix “A” used in the old USDA scheme.  

Classification 281.9 is for societies, boards and institutions in the field of agricultural economics.  It is the old USDA library classification scheme.  If there is a prefix A in front of 281.9, it means this is a USDA publication.  It should not be removed."

**Comment by AmandaZXu-USDA** [comment]
@CECSpecialistI @AdamSchiff @GordonDunsire  and all

Thanks a lot for the posting of my responses.  I hope this version that I posted here is clearer than the email responses.  NAL Call numbers can be grouped into the following categories;

1. Prior 1965: NAL Classification Scheme: 
i.	USDA publication:  A1-A517, e.g. A280.P69 1961, A281.005.Ag83, A569.9.C73C 1959
ii.	Other publications: [1-9].A-Z, where [1-9] as number ranging from 1-517, followed by cutter number for the author or organization,  e.g. 42.C88, where C is for author cutter; 281.9.N27.no.154, where N27 is for Nebraska; 281.9.P38 no.247, where P38 is for Pennsylvania

2.	After 1965, LC Classification Scheme: 
i.	USDA publication: aA-aZ; 
ii.	Juvenile literature: jA-jZ, e.g. jTX652.F36 2005, jRA448.5.15.P46785 2015;
iii.	Rare Book: R added to the end of call number: e.g. 463.W13 Ed.3 R , 409.5.W67 R 
iv.	Other publications: A[1-9]-Z[1-9], e.g.  QL49.M56 1886
3.	Accession number – call number prefix for GRAPHIC, KIT, etc. e.g. Graphic.no.592, Kit.no.681
4.	None classification scheme, e.g. NOT KEPT, NOT KEPT WITHDRAWN 
5.	After 1989, “Ov” added to the end of call number for oversize, and “Fo” added to the end of call number for folio
6.	Serials publications:  Serial-like items have retained the old call number whenever possible; thus, both NAL classification scheme and LC classification scheme are still in use.

Please let me know if you have any questions.

Best regards,

Amanda

**Comment by CECSpecialistI** [comment]
@szapoun Same question about the review status for this tag.

**Comment by CECSpecialistI** [comment]
@cspayne did we figure out how to code this yet? Is it clear what needs to happen to distinguish serial-like items from others, and which scheme to apply where? I think the spreadsheet has the transformation notes swapped for serial-like and non-serial-like items, and it isn't clear from the spreadsheet how we propose distinguishing them.

**Comment by cspayne** [comment]
@CECSpecialistI This never got coded, it looks like we got an answer after I left. I'm unsure if the spreadsheet is up to date and ready to code or not.

**Comment by CECSpecialistI** [comment]
I think it depends on whether there is clear enough direction for coders to determine what a "serial-like item" is. is this enough of a condition to go on?

**Comment by CECSpecialistI** [comment]
Meeting guidance: Need a work-around for scheme of nomen to indicate this number comes from NAL.

**Comment by tmqdeborah** [comment]
Some Nomen element possibilities, in lieu of being able to determine which classification scheme was used.
- [assigned by corporate body](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-87d99d0d-bb88-34e6-8bdd-19fb6d8fc9af)
- [context of use](https://access.rdatoolkit.org/Content/ContentById/d085a9a2-204f-4aa7-91d4-6d2af8d0eb82)
- [note on nomen](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-8e3608cc-db30-3aeb-9cb6-df30f5d0b99f)

"Assigned by corporate body" ([n/P80075](http://rdaregistry.info/Elements/n/P80075)) seems to me to best reflect the MARC definition of the field: "Classification or call number that is assigned by the National Agricultural Library (U.S.)" 

Name taken from [LCNAF](https://lccn.loc.gov/n80009063):
- datatype; structured
- or object; IRI

Result:
Assigned by corporate body : National Agricultural Library (U.S.)

**Comment by CECSpecialistI** [comment]
> Some Nomen element possibilities, in lieu of being able to determine which classification scheme was used.
> 
> * [assigned by corporate body](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-87d99d0d-bb88-34e6-8bdd-19fb6d8fc9af)
> * [context of use](https://access.rdatoolkit.org/Content/ContentById/d085a9a2-204f-4aa7-91d4-6d2af8d0eb82)
> * [note on nomen](https://access.rdatoolkit.org/Content/Index?externalId=en-US_ala-8e3608cc-db30-3aeb-9cb6-df30f5d0b99f)
> 
> "Assigned by corporate body" ([n/P80075](http://rdaregistry.info/Elements/n/P80075)) seems to me to best reflect the MARC definition of the field: "Classification or call number that is assigned by the National Agricultural Library (U.S.)"
> 
> Name taken from [LCNAF](https://lccn.loc.gov/n80009063):
> 
> * datatype; structured
> * or object; IRI
> 
> Result: Assigned by corporate body : National Agricultural Library (U.S.)

I agree that assigned by corporate body is the best property to use. I'll update the spreadsheet and pass this along to the transform team.

**Comment by CECSpecialistI** [comment]
Oops. We weren't minting nomens here, we are minting concepts. Using SKOS. I think the best we can do here is [skos:note](http://www.w3.org/2004/02/skos/core#note).
